File: tests/resources/test_codegen/template.go
package main

import (
	"bufio"
	"os"
	"strconv"
)

func solve(${formal_arguments}) {

}

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	const initialBufSize = 4096
	const maxBufSize = 1000000
	scanner.Buffer(make([]byte, initialBufSize), maxBufSize)
	scanner.Split(bufio.ScanWords)
	${input_part}
	solve(${actual_arguments})
}

File: tests/resources/test_codegen/template.swift
import Foundation

func solve({{ formal_arguments }}) {

}

func main() {
    func readString() -> String { "" }
    func readInt() -> Int { 0 }
    func readDouble() -> Double { 0 }
    {{input_part}}
    _ = solve({{ actual_arguments }})
}

main()
